Packaging/resources/devilutionx.mpq.txt

```diff
@@ -0,0 +1,10 @@
+data\health.pcx
+data\healthbox.pcx
+data\resistance.pcx
+data\xpbar.pcx
+Gendata\Cutportlw.pcx
+Gendata\Cutportrw.pcx

```

Comment:
despite calling with upper case all files are looked up in lowercase on fs

```suggestion
gendata\cutportlw.pcx
gendata\cutportrw.pcx
```

Reply:
I made this change last night without really thinking about it, but perhaps I should have asked. MPQ Editor actually has no trouble finding the files when using a listfile with mixed casing. Is this just an implementation detail for MPQ Editor to make the filenames case insensitive, or is that how the MPQ format works in general? Wouldn't it be better to use the mixed-case names if it doesn't cause problems for MPQ readers?